src/install.ts

```diff
@@ -11,19 +12,21 @@ function getPlatform() {
     return platform;
   } else if(platform === 'linux') {
     return 'linux-glibc';
+  } else if(platform === 'win32') {
+    return 'windows'
   } else {
     throw "prisma-fmt: Unsupported OS platform: " + platform;
   }
 }
 
 // Gets the download URL for a platform
-function getFmtDownloadUrl(platform) {
-  return "https://s3-eu-west-1.amazonaws.com/prisma-native/alpha/latest/" + platform + "/prisma-fmt.gz";
+function getFmtDownloadUrl(platform, basename: string) {
+  return "https://s3-eu-west-1.amazonaws.com/prisma-native/alpha/latest/" + platform + "/" + basename + ".gz";
```

Comment:
Please adapt the new binary paths as specified in https://github.com/prisma/specs/tree/master/binaries#s3-url-schema. Thanks!

Reply:
I've just updated it.